Oneirocom/Magick
# Run cicd on the development branch too.

Avoid releases on the dev branch.

## What Changed:

I added the electron cicd for the development branch.

Only release on the production branch.

Development branch creates temporary artifacts.

TODO: move to capacitor

## How to test:

How can the reviewer test your changes and validate that they work as described?

Download each electron build and see if they run. Try to cover windows, linux and mac.

## Additional information:

Any other information that might be useful while reviewing.

Why capacitor? Capacitor allows native apps for the mobile platform.
